tests/main/degraded/task.yaml

```diff
@@ -22,6 +22,13 @@ execute: |
     . "$TESTSLIB"/systemd.sh
     wait_for_service "multi-user.target"
 
+    case "$SPREAD_SYSTEM" in
+        opensuse-tumbleweed-*)
+            systemctl mask systemd-vconsole-setup.service
```

Comment:
is mask needed?

Reply:
Weirdly, yes. Without the mask the reset failed would not clear the state of the system.